indexer/Cargo.toml

```
@@ -31,8 +31,8 @@ tracing = "0.1.13"
 tracing-subscriber = "0.2.4"
 uint = { version = "0.8.3", default-features = false }
 
-near-jsonrpc-client = "0.4.0-beta.0"
-near-lake-framework = "0.5.2"
+near-jsonrpc-client = "0.5.0"
+near-lake-framework = "0.7.0"
```

Don't forget to update the Lake Framework
```suggestion
near-lake-framework = "0.7.1"
```